data/json/overmap/overmap_terrain/overmap_terrain.json

```diff
@@ -284,7 +284,7 @@
     "color": "light_gray",
     "see_cost": 2,
     "mondensity": 2,
-    "spawns": { "group": "GROUP_WASP_GUARD", "population": [ 5, 10 ] }
+    "spawns": { "group": "GROUP_WASP_GUARD", "population": [ 3, 8 ], "chance":100 }
```

Comment:
`"chance": 100` (note space after the colon, BTW) is needed when there's only one thing spawning?

Reply:
Yyyup. That goes on the monster spawning weirdness list.

Reply:
Thank you, I'll have to remember that... sigh.